### PR TITLE
Get pixel color of full camera width

### PR DESCRIPTION
**Pull request by hansman**
- Get pixel color of full camera width (`onDraw`)
- Default frame width to camera width to start with (`setup`)
- Allow frame width to take values from 0 to camera width (`onMouseMove`)

### CODE REVIEW

File: src/ofApp.cpp [hunk 1]
```diff
@@ -7,21 +7,23 @@ void ofApp::setup(){
 	
 	camWidth 		= 640;	// try to grab at this size. 
 	camHeight 		= 480;
+    
+    frameWidth = camWidth;
```
Comment (uniphil) [inline src/ofApp.cpp:11]: nice

File: src/ofApp.cpp [hunk 2]
```diff
@@ -90,7 +92,7 @@ void ofApp::keyReleased(int key){
 
 //--------------------------------------------------------------
 void ofApp::mouseMoved(int x, int y ){
-    frameWidth = abs(320 - x) + 1;
```
Comment (uniphil) [inline src/ofApp.cpp:93]: should this be `abs(camWidth / 2 - x) + 1`?

**Review by uniphil — APPROVED** [review]

File: src/ofApp.cpp (hunk 2 above)
Comment (hansman) [reply src/ofApp.cpp:93]: Don't know. If divided by two you can never scan the full web cam width.

Comment (uniphil) [reply src/ofApp.cpp:93]: right! I think this:
`abs(camWidth / 2 - x) * 2 + 1` will do what I intended -- distance from centre, but follow mouse to the full width on both sides. But as you said, it's totally whatever UX you like.

**Pull request merged**